src/htm/algorithms/SDRClassifier.cpp

```diff
@@ -67,6 +66,12 @@ PDF Classifier::infer(const SDR & pattern) const {
   return probabilities;
 }
 
+void Classifier::learn(const SDR &pattern, UInt category)
```

Comment:
what do these do? Maybe add doc that it's convenience for learning a single category (if I'm right about its use?)